code/src/components/Albumshero.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Covers from './Covers';
+import Album from './Album';
+import Artist from './Artist';
+import AlbumOverlay from './AlbumOverlay';
+
+const Albumshero = (props) => {
+  console.log(props.data);
+  return (
+    <>
+     <div class="musicCard">
+     <a class ="album-link" href={props.data.external_urls.spotify} target="_blank" rel="noopener noreferrer">
```

Comment:
It is a general convention to separate lines if more than 2 attributes have used.

For example: 

> <img

  src={props.album.images[1].url}
  className="cover-img"
  alt={props.album.name}
/>